QQE-378.md

```diff
@@ -0,0 +1,43 @@
+# QQE-378 - HTTP module / REST use-cases - Development topic compression Brotli4j
+## Links
+Jira : [https://issues.redhat.com/browse/QQE-378](https://issues.redhat.com/browse/QQE-378)
+
+PR:  TODO
+## Scope of the testing
```

Comment:
I think your objection needs to be map whether it works in native (probably not, but we need to now the state)